alphagov/pay-cardid
# PP-682 Adding new relic

## WHAT
_A brief description of the pull request:_
 * Adding new relic Jar and respective configuration

with @PauloPortugal

## HOW 
_Steps to test or reproduce:_
* Run `msl up` in pay-cardid project
* Run `docker exec -ti pay_cardid_1 bash` and do a `ls` to see if the newrelic folder exists.

## WHO
_Who should review this pull request:_

- [ ] WebOps
